src/platform/packages/shared/kbn-resizable-layout/index.ts

```diff
@@ -7,9 +7,6 @@
  * License v3.0 only", or the "Server Side Public License, v 1".
  */
 
-import { withSuspense } from '@kbn/shared-ux-utility';
-import { lazy } from 'react';
-
 export { ResizableLayoutMode, ResizableLayoutDirection } from './types';
 export type { ResizableLayoutProps } from './src/resizable_layout';
-export const ResizableLayout = withSuspense(lazy(() => import('./src/resizable_layout')));
```

Comment:
The lazy export was unnecessary since this lives in a package anyway, and was just slowing Discover down by loading an additional async module on initialization.